modules/viewing-security-information-in-red-hat-developer-hub.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies:
+//
+// *configuration/configuring-and-integrating-the-rhacs-plugin-with-red-hat-developer-hub.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="viewing-security-information-in-red-hat-developer-hub_{context}"]
+= Viewing security information in Red Hat Developer Hub
+
+By configuring and integrating the {rh-rhacs-first} plugin with {rh-rhdh-first}, you can access vulnerability data, assess risks, and take proactive security actions without leaving the {rh-rhdh} environment.
+
+.Prerequisites
+
+* You have enabled the {product-title-short} plugin installation in {rh-rhdh}.
++
+For more information, see link:https://docs.redhat.com/en/documentation/red_hat_developer_hub/{rhdh-latest-version}/html/installing_and_viewing_plugins_in_red_hat_developer_hub/rhdh-installing-rhdh-plugins_title-plugins-rhdh-about#con-install-dynamic-plugin-helm_rhdh-installing-rhdh-plugins[Installing dynamic plugins using the Helm chart].
+
+.Procedure
+
+. Create an `app-config.yaml` file with the `proxy` and `acs` stanza by using the following content:
++
+[source,yaml]
+----
+# ...
+proxy:
+  endpoints:
+    /acs:
+      credentials: dangerously-allow-unauthenticated
+      target: ${ACS_API_URL}
+      headers:
+        authorization: Bearer ${ACS_API_KEY}
+acs:
+  acsUrl: ${ACS_API_URL}
+# ...
+----
+
+. To enable the {product-title-short} plugin, perform the following steps:
```

Comment:
```suggestion
. Enable the {product-title-short} plugin:
```

Reply:
It's a lead-in sentence in either case.

Reply:
Oh, wait, my mistake. It's not a lead-in sentence to a procedure--it's just a step. https://www.ibm.com/docs/en/ibm-style?topic=format-procedures#writing-steps

Both the original and the suggestion remain acceptable to ISG, though. It's just word golf. 🤷